content/en/mobile_testing/mobile_tests/steps.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Mobile Test Steps
+kind: documentation
+description: Learn how to automatically record and manually set steps in a mobile test recording.
+further_reading:
+- link: "/mobile_testing/mobile_tests/"
+  tag: "Documentation"
+  text: "Learn about Synthetic mobile tests"
+---
+
+## Overview
+
+Steps are a series of actions that you can record for a mobile test and edit or build on. To define the steps you want your mobile test to execute, either directly record them with the mobile test recording or add them manually. 
+
+## Automatically recorded steps
+
+Once you click **Start Recording**, the Datadog mobile test recorder automatically launches a mobile application and records steps on your device.
+
+## Manually added steps
+
+You can manually add and arrange steps on the left side of the mobile test recording.
+
+### Assertion
+
+Assertions allow you to validate that your browser test is in the state you expect it to be in at any point of a simulated user journey. 
+
+To confirm your test ends in an expected state, you must end your mobile tests with an **assertion**.
+
+// replace
+
+{{< img src="synthetics/browser_tests/browser_test_assertions.png" alt="Options for assertions in a browser test step" style="width:70%;" >}}
+
+Some assertions validate the active page, the page the user last interacted with, such as a **click** or an **assertion** on a page element.
+
+To create a step, select an assertion type:
+
+{{< tabs >}}
+{{% tab "Test An Element On The Active Screen" %}}
+
+#### Test an element's content
+
+Create this assertion step to have your mobile test select a page element and check if it contains a specific value. 
+
+{{% /tab %}}
+{{% tab "Test Active Screen Content" %}}
+
+#### Test that some text is present on the active screen
+
+Create this assertion step to have your browser test confirm that the text you specified in the `Value` field is present on the current page being recorded.
+
+#### Test that some text is not present on the active screen
+
+Create this assertion step to have your browser test confirm that the text you specified in the `Value` field is **not** present on the current page being recorded.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Special actions
```

Comment:
Technically but since it's a header, we use sentence case (Vale linter rule will pop up for title case in headings: https://github.com/DataDog/datadog-vale/blob/main/styles/Datadog/headings.yml#L6.